src/utils/beneficial.owner.individual.ts

```diff
@@ -184,21 +185,28 @@ export const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDat
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
   data[CeasedDateKey] = reqBody["is_still_bo"] === '0' ? mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys) : {};
 
+  setBoNocData(data);
```

Comment:
Logic placed in a separate local function to address the Sonar complexity violation.